### DISCUSSION TITLE
"phase_offset_distribution" == "grid"

### DISCUSSION THREAD

**Discussion by ShirleyMgit** [RatInABox-Lab/RatInABox#112] [Q&A] [answered]
Is it working? it does not work for me...

**Comment by TomGeorge1234** [comment, marked as answer]
Hi this wouldn't work. Instead `phase_offset_distribution` should be used to pass in the _distribution_ from which phase offsets will be sampled. The parameters of this distribution should be passed in as a tuple under `phase_offset`.  Currently allowed distributions are 
    
```  
    - uniform ------------------------------- (low, high) or just a single param p which gives (0.5*p, 1.5*p)
    - rayleigh ------------------------------ (scale)
    - normal -------------------------------- (loc, scale)
    - logarithmic --------------------------- (low, high)
    - delta --------------------------------- (the_single_value)
    - modules ------------------------------- (module1_val, module2_val, module3_val, ...)
    - truncnorm ----------------------------- (low, high, loc, scale)
```

Alternatively make your own list/array (not a tuple)  of phase offsets and pass them in under `phase_offset`. Hope this helps for more info see the doctoring for the grid cells class.

**Comment by TomGeorge1234** [comment]
Assuming all is clear now and closing
